Conceptpower+Spring/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -205,8 +205,8 @@ function prepareMergeConcept(conceptId) {
     }
 }
 
-var createWrapper = function(word, pos, conceptList, description, conceptType, wrapperId) {
-  window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?word=' + word + '&selectedConceptList=' + conceptList + '&description=' + description + '&selectedType=' + conceptType + '&wrapperids=' + wrapperId;
+function createWrapper(word, wrapperId) {
+  window.location = '${pageContext.servletContext.contextPath}/auth/conceptlist/addconceptwrapper?word=' + word + '&wrapperids=' + wrapperId;
```

Comment:
you don't need the word, do you? just the id.

Reply:
Removed word. Just sent the wrapper id.